srcpkgs/SDL2_image/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'SDL2_image'
 pkgname=SDL2_image
-version=2.0.5
-revision=2
+version=2.6.2
+revision=1
 build_style=gnu-configure
-configure_args="--disable-static --disable-jpg-shared
- --disable-png-shared --disable-webp-shared --disable-tif-shared"
+configure_args="--disable-static --disable-jpg-shared --disable-png-shared
+ --disable-stb-image --disable-tif-shared --disable-jxl-shared
+ --disable-webp-shared"
 hostmakedepends="pkg-config"
 makedepends="libpng-devel tiff-devel SDL2-devel libwebp-devel"
 short_desc="Load images as SDL surfaces (SDL 2.x)"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
 license="Zlib"
 homepage="http://www.libsdl.org/projects/SDL_image/"
```

Comment:
```suggestion
homepage="https://github.com/libsdl-org/SDL_image"
changelog="https://raw.githubusercontent.com/libsdl-org/SDL_image/main/CHANGES.txt"
```

Reply:
Thanks for the changelog, wasnt aware of that one !!!